inc/class-db.php

```diff
@@ -64,7 +66,7 @@ public function query( $query ) {
 			return $result;
 		}
 
-		$i = count( $this->queries ) - 1;
+		$i = count( (array) $this->queries ) - 1;
```

Comment:
The error message says that `$this->queries` is actually `null`. We should understand why it can be `null` in some cases and most likely return early rather than just cast it as an array here. `null` is not the same as `[ null ]` which will have a `count()` of 1.